client/my-sites/checkout/src/components/wp-checkout.tsx

```diff
@@ -429,6 +429,21 @@ export default function WPCheckout( {
 			? translate( 'Continue to payment', { textOnly: true } )
 			: translate( 'Continue', { textOnly: true } );
 
+	/* Include a condition for your use case here if you want to show a specific nudge in the checkout sidebar */
+	const renderCheckoutSidebarNudge = () => {
```

Comment:
A function that returns markup is, effectively, a component. Could we make this a standalone component instead of having it be an inline function? That will make its dependencies more clear as well as making sure that we don't end up with any weird rendering behavior like you fixed in https://github.com/Automattic/wp-calypso/pull/86077. When the function is inline it will be recreated on every render, which may prevent React from knowing that it's the same component.